tests/hp/fe_nothing_12.cc

```diff
@@ -57,7 +57,9 @@ void test ()
   std::vector<unsigned int> sub(dim, 1);
   sub[dim-1] = 2;
 
-  GridGenerator::subdivided_hyper_rectangle (triangulation, sub, Point<dim>(), Point<dim>());
+  Point<dim> p1;
+  Point<dim> p2(1.0,2.0);
```

Comment:
I find this code a bit confusing because you use a generic template `<dim>` here but then hardcode the Point constructor to `dim=2`. But I guess this is fine anyway.